service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/CloneWorkspaceFlight.java

```diff
@@ -58,68 +32,7 @@ public CloneWorkspaceFlight(FlightMap inputParameters, Object applicationContext
 
     addStep(new CreateIdsForFutureStepsStep());
 
-    // Only create a cloud context if the source workspace has a cloud context
-    Optional<GcpCloudContext> gcpCloudContextOptional =
```

Comment:
Cloud context creation now happens before this flight is called, when we create the destination workspace.